src/bio/bio_xstream.c

```diff
@@ -380,51 +157,24 @@ bio_nvme_init(const char *nvme_conf, int shm_id, int mem_size,
 		return 0;
 	}
 
-	fd = open(nvme_conf, O_RDONLY, 0600);
-	if (fd < 0) {
-		D_WARN("Open %s failed, skip DAOS NVMe setup "DF_RC"\n",
-		       nvme_conf, DP_RC(daos_errno2der(errno)));
-		nvme_glb.bd_nvme_conf = NULL;
-		return 0;
-	}
-	close(fd);
-
-	rc = smd_init(db);
-	if (rc != 0) {
-		D_ERROR("Initialize SMD store failed. "DF_RC"\n", DP_RC(rc));
-		return rc;
-	}
-
-	nvme_glb.bd_nvme_conf = spdk_conf_allocate();
-	if (nvme_glb.bd_nvme_conf == NULL) {
-		D_ERROR("Failed to alloc SPDK config\n");
-		rc = -DER_NOMEM;
-		goto free_cond;
-	}
-
-	rc = spdk_conf_read(nvme_glb.bd_nvme_conf, nvme_conf);
-	if (rc != 0) {
-		rc = -DER_INVAL; /* spdk_conf_read() returns -1 */
-		D_ERROR("Failed to read %s, "DF_RC"\n", nvme_conf, DP_RC(rc));
-		goto free_conf;
-	}
+	 rc = smd_init(db);
+	 if (rc != 0) {
+	 	D_ERROR("Initialize SMD store failed. "DF_RC"\n", DP_RC(rc));
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs